workloads/network-perf-v2/run.sh

```diff
@@ -39,5 +39,5 @@ if [ $run -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" ES_SERVER="$ES_SERVER" ../../utils/index.sh
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```

Comment:
hm, WORKLOAD in this case is smoke.yaml or full.yaml which isn;t all to useful, we might just want to prefix `k8s-netperf`

Reply:
or maybe `netperf`? thoughts?

Reply:
Adding it as `k8s-netperf` according to our tool name.